scripts/repository.lic

```diff
@@ -1206,7 +1227,7 @@ if cmd[0] =~ /^list$/i
     echo "error: extra words on command line: #{cmd[1..-1].join(' ')}"
     exit
   end
-  respond "\n#{format_list.call(filter_list.call(get_list.call))}\n\n"
+  puts "\n#{format_list.call(filter_list.call(get_list.call), 'info')}\n\n"
```

Comment:
do NOT use `puts` as it has known issues with terminal based FEs. In order to send actual XML, change the `respond` to `_respond`